packages/@aws-cdk/aws-apprunner-alpha/lib/vpc-ingress-connection.ts

```diff
@@ -0,0 +1,168 @@
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as cdk from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import { IService } from './service';
+import { CfnVpcIngressConnection } from 'aws-cdk-lib/aws-apprunner';
+
+/**
+ * Properties of the AppRunner VPC Ingress Connection
+ */
+export interface VpcIngressConnectionProps {
+  /**
+   * The name for the VPC Ingress Connection.
+   *
+   * @default - a name generated by CloudFormation
+   */
+  readonly vpcIngressConnectionName?: string;
+
+  /**
+   * The service to connect.
+   */
+  readonly service: IService;
+
+  /**
+   * The VPC for the VPC Ingress Connection.
+   */
+  readonly vpc: ec2.IVpc;
+
+  /**
+   * The VPC Interface Endpoint for the VPC Ingress Connection.
+   */
+  readonly interfaceVpcEndpoint: ec2.IInterfaceVpcEndpoint;
+}
+
+/**
+ * Attributes for the App Runner VPC Ingress Connection
+ */
+export interface VpcIngressConnectionAttributes {
+  /**
+   * The Amazon Resource Name (ARN) of the VPC Ingress Connection.
+   */
+  readonly vpcIngressConnectionArn: string;
+
+  /**
+   * The name of the VPC Ingress Connection.
+   */
+  readonly vpcIngressConnectionName: string;
+
+  /**
+   * The domain name associated with the VPC Ingress Connection resource.
+   */
+  readonly domainName: string;
+
+  /**
+   * The current status of the VPC Ingress Connection.
+   */
+  readonly status: string;
```

Comment:
The followings are status. 

`AVAILABLE | PENDING_CREATION | PENDING_UPDATE | PENDING_DELETION | FAILED_CREATION | FAILED_UPDATE | FAILED_DELETION | DELETED`

https://docs.aws.amazon.com/apprunner/latest/api/API_VpcIngressConnection.html

I think this item should not be enum because this item is an attribute not props.
Please let me know if I misunderstood.

Reply:
Good point, ya let's keep it as a string to match with the CloudFormation type.